llm/gemini_adapter.py

```
@@ -0,0 +1,123 @@
+            response_schema=request.response_schema,
+            response_mime_type="application/json",
+        )
+
+        start = time.perf_counter()
+        try:
+            response = client.models.generate_content(
+                model=self._model,
+                contents=request.user_prompt,
+                config=config,
+            )
+        except Exception as err:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)